instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py

```diff
@@ -270,10 +263,11 @@ def test_dsl_create(self, request_mock):
         client = Elasticsearch()
         Article.init(using=client)
 
-        spans = self.get_ordered_finished_spans()
+        spans = self.get_finished_spans()
         self.assertEqual(2, len(spans))
-        span1, span2 = spans
-        self.assertEqual(span1.name, "Elasticsearch/test-index")
```

Comment:
Why delete this check?

Reply:
LMK if that makes sense. I can follow up with another PR.